bindings/cpp/source/portaudiocpp/CMakeLists.txt

```
@@ -0,0 +1,26 @@
+find_package(ASIO)
+if(ASIO_FOUND)
+    set(portaudio-cpp-sources-asio AsioDeviceAdapter.cxx PARENT_SCOPE)
+else()
+    set(portaudio-cpp-sources-asio PARENT_SCOPE)
+endif()
+
+set(portaudio-cpp-sources
+    ${CMAKE_CURRENT_SOURCE_DIR}/BlockingStream.cxx
```

```suggestion
    BlockingStream.cxx
```
unneeded when listed in parent directory's CMakeLists.txt